1882 - Fixed selected all was not working with dropdown (#1936)
infor-design/enterprise: docs/CHANGELOG.md

- `[Datagrid]` Fixed an issue where lookup filterConditions were not rendering. ([#1873](https://github.com/infor-design/enterprise/issues/1873))
- `[Datagrid]` Fixed an issue where filtering was missing translation. ([#1900](https://github.com/infor-design/enterprise/issues/1900))
- `[Dropdown]` Changed the way dropdowns work with screen readers to be a collapsible listbox.([#404](https://github.com/infor-design/enterprise/issues/404))

> - `[Dropdown]` Fixed an issue where multiselect dropdown unchecking "Select All" was not getting clear after close list with Safari browser.([#1882](https://github.com/infor-design/enterprise/issues/1882))

Reviewer: ```suggestion
- `[Dropdown]` Fixed an issue in multiselect where when unchecking "Select All" it was not getting cleared after closing list when using Safari browser.([#1882](https://github.com/infor-design/enterprise/issues/1882))
```
